File: Domains/Base/Workspace/CSharp/Domain/Base/Workspace/Data/ErrorResponse.cs
namespace Allors.Workspace.Data
{
    public interface ErrorResponse
    {
        bool hasErrors { get; set; }

        string errorMessage { get; set; }

        string[] versionErrors { get; set; }

        string[] accessErrors { get; set; }

        string[] missingErrors { get; set; }

        PullResponseDerivationError[] derivationErrors { get; set; }
    }
}

File: Domains/Test/Workspace/Typescript/Angular/src/allors/domain/base/data/responses/ErrorResponse.ts
import { Response } from "./Response";
import { PullResponseDerivationError } from "./PullResponseDerivationError";

export interface ErrorResponse extends Response {
    hasErrors: boolean;
    errorMessage?: string;
    versionErrors?: string[];
    accessErrors?: string[];
    missingErrors?: string[];
    derivationErrors?: PullResponseDerivationError[];
}
